Atomic/scan.py

```diff
@@ -176,8 +176,9 @@ def gen_containers():
                                        if item['Id'] == self.get_input_id(scan_input)), None))
                 docker_object = docker_object if docker_object is not None \
                     else (next((item for item in images if item['Id'] == self.get_input_id(scan_input)), None))
-                docker_object['input'] = scan_input
-                scan_list.append(docker_object)
+                if docker_object is not None:
```

Comment:
With the fix it gives an error that the name was not found:

[root@rhel-base atomic]# atomic scan asldkfjasdlfj
Unable to associate 'asldkfjasdlfj' with an image or container
[root@rhel-base atomic]# 

instead of the previous behavior:

[root@rhel-base docker]# atomic scan asldkfjasdlfj
    'NoneType' object does not support item assignmentn[root@rhel-base docker]#

If you follow the code, it basically says:
docker_obj = (search the image list, default None)
docker_obj = (search the container list, default None) if docker_object is not None
there was no clause where it handled the condition where both of those statements resulted in None.